script/import_db.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+cd `dirname $0`/..
+
+DATABASE=READWRITE
+DIRECTORY=./dumps
+
+source ./admin/functions.sh
+
+if ! script/database_exists $DATABASE; then
+    count=`ls -l $DIRECTORY/mbdump*.tar.bz2 2>/dev/null | wc -l`
+    if [ $count -ne 0 ]; then
+        echo "Creating Database and Importing Dump"
```

Comment:
I'd remove all these `echo`s, since they don't say anything additional, or otherwise fix the capitalization. :)